app/build.gradle

```diff
@@ -45,8 +45,17 @@ android {
     // Map for the version code that gives each ABI a value.
     def abiCodes = ['armeabi-v7a': 1, 'arm64-v8a': 2, 'x86': 3, 'x86_64': 4]
 
+    // Enumerate translated locales
+    def translatedLocales = []
```

Comment:
i think it's better to have this initialized with "en" to show it is the default/fallback language.